Resources/config/services.yml

```diff
@@ -1,9 +1,8 @@
 
 services:
-  happyr.linkedin:
-    class: Happyr\LinkedIn\LinkedIn
+  Happyr\LinkedIn\LinkedIn:
```

Comment:
Why do we need to update the service id?

Reply:
http://take.ms/PtQTq
https://symfony.com/doc/current/service_container/3.3-di-changes.html

Reply:
And also few log messages about deprecating:
http://take.ms/qYJLx
http://take.ms/kYEyP

Reply:
Yeah, Im very aware of those changes. Could you please just add the quotes around the parameters. 

And then to fix the autowire thing we need to do something like: 

```
services: 
  Happyr\LinkedIn\LinkedIn:
    alias: happyr.linkedin
    public: false
```

Reply:
Of course, there must be backward compatibility! Thank you for remind.